src/pal/inc/rt/palrt.h

```diff
@@ -561,6 +571,21 @@ enum VARENUM {
 };
 
 typedef struct tagVARIANT VARIANT, *LPVARIANT;
+typedef struct tagSAFEARRAY SAFEARRAY;
+
+#if !defined(_FORCENAMELESSUNION)
```

Comment:
Is this ifdef needed? Will nameless unions break anything if they are used everywhere?